src/foam/u2/crunch/wizardflow/lite/CapableCreateWizardletsAgent.js

```diff
@@ -24,31 +28,74 @@ foam.CLASS({
 
   methods: [
     async function execute() {
-      var wizardlets = [];
       var capable = this.capable;
+      this.wizardlets = this.createWizardletsFromPayloads(capable.capablePayloads);
+      console.log('CAPABLE', capable);
+      console.log('WIZARDLETS', this.wizardlets);
+    },
+    function createWizardletsFromPayloads(payloads) {
+      debugger;
+      var newWizardlets = [];
+      for ( let i = 0 ; i < payloads.length ; i++ ) {
+        let capablePayload = payloads[i];
+        let wizardlet = this.createWizardletFromPayload(capablePayload);
```

Comment:
So this only gets fired once?

Reply:
`createWizardletFromPayload` once for each Payload, but also once for each prerequisite payload